fastlane/lib/fastlane/actions/dotenv.rb

```diff
@@ -0,0 +1,91 @@
+require 'dotenv'
+
+module Fastlane
+  module Actions
+    class DotenvAction < Action
+      def self.run(params)
+        env_file = ".env.#{params[:name]}"
+        path = params[:path]
+        overload = params[:overload]
+        fail_if_missing = params[:fail_if_missing]
+
+        if path
+          env_file = File.join(path, env_file)
+        else
+          base_path = Fastlane::Helper::DotenvHelper.find_dotenv_directory
```

Comment:
Perhaps move this to the path's default value in the `available_options`, WDYT? Then I think we can make it non-null too (users would still be able to omit it and it'd default to this dynamic value)

Reply:
Oh yeah lol... that makes A LOT more sense 😅